.travis.yml

```diff
@@ -1,7 +1,18 @@
 language: c
-install: wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-opam.sh
-script: bash -ex .travis-opam.sh
-dist: trusty
+sudo: false
+services:
+  - docker
+install: wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-docker.sh
+script: bash ./.travis-docker.sh
 env:
-  - PACKAGE="xenctrl" OCAML_VERSION=4.01
-  - PACKAGE="xenctrl" OCAML_VERSION=latest
+ global:
+   - PACKAGE="xenctrl"
+ matrix:
+   - DISTRO=debian-stable OCAML_VERSION=4.01.0
+   - DISTRO=debian-testing OCAML_VERSION=4.02.3
+   - DISTRO=debian-unstable OCAML_VERSION=4.03.0
+   - DISTRO=ubuntu-12.04 OCAML_VERSION=4.02.3
```

Comment:
I made a fresh alpine install to undestand the problem. It ships xen-dev-4.5 after all.

To make it work on alpine I had to modify `configure` as follows
```
- D=$(mktemp -d /tmp/configure.XXXXX)
+ D=$(mktemp -d /tmp/configure.XXXXXX)
```
Then I had to fake the `arch` command because it's not shipped in `coreutils`, and I did not know where to find it for alpine (this might be not necessary in travis, I don't know). I did
```
echo "#!/bin/bash\necho x86_64" > /usr/local/bin/arch
chmod +x /usr/local/bin/arch
```
and this made the compilation complete properly.